js/spoiler.js

```diff
@@ -0,0 +1,13 @@
+function getSpoilers() {
```

Comment:
this file and function will be unneeded if you remove duplication with all this classname toggle stuff